test/controllers/content_items_controller_test.rb

```diff
@@ -137,15 +137,13 @@ class ContentItemsControllerTest < ActionController::TestCase
 
     content_store_has_item(content_item['base_path'], content_item)
 
-    with_variant EducationNavigation: "A" do
```

Comment:
What's the issue with using `with_variant` here? Is it something we should change in the A/B gem?